src/components/ui/Walkthrough.js

```diff
@@ -0,0 +1,74 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+class Walkthrough extends PureComponent {
+
+  static propTypes = {
+    title: PropTypes.string.isRequired,
```

Comment:
Agree, it doesn't follow the format, but its easier to read and to know what props to expect inside the component. What we're doing in the mobile app is, changing the location of the propTypes when editing an existing component. I think this would be the best approach.